molecule/default/tests/conftest.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+# ======================================================================================================================
+# Imports
+# ======================================================================================================================
+import pytest
+import openstack
```

Comment:
Quick question: This 'openstack' can be imported after installing 'openstacksdk' on rpc-openstack-system-tests?

Reply:
@phongdly Correct.